src/components/common/Stepper.vue

```diff
@@ -27,153 +27,7 @@ import { GetterIF } from '@/interfaces'
-  //         Stepper and Actions can use it.
-  //
-
-  // Step configuration for BCOMP
-  private get StepsBC (): Array<any> {
-    return [{
-      id: 'step-1-btn',
-      icon: 'mdi-domain',
-      text: 'Define Your\nCompany',
-      to: '/define-company',
-      disabled: this.isBusySaving,
-      valid: this.step1Valid
-    },
-    {
-      id: 'step-2-btn',
-      icon: 'mdi-account-multiple-plus',
-      text: 'Add People\nand Roles',
-      to: '/add-people-roles',
-      disabled: this.isBusySaving,
-      valid: this.step2Valid
-    },
-    {
-      id: 'step-3-btn',
-      icon: 'mdi-sitemap',
-      text: 'Create Share\nStructure',
-      to: '/share-structure',
-      disabled: true,
-      valid: this.step3Valid
-    },
-    {
-      id: 'step-4-btn',
-      icon: 'mdi-handshake',
-      text: 'Incorporation\nAgreement',
-      to: '/incorp-agreement',
-      disabled: true,
-      valid: this.step4Valid
-    },
-    {
-      id: 'step-5-btn',
-      icon: 'mdi-text-box-check-outline',
-      text: 'Review\nand Confirm',
-      to: '/review-confirm',
-      disabled: this.isBusySaving,
-      valid: this.isApplicationValid
-    }]
-  }
-
-  // Step configuration for COOP
-  private get StepsCP (): Array<any> {
-    return [{
-      id: 'step-1-btn',
-      icon: 'mdi-domain',
-      text: 'Define Your\nCompany',
-      to: '/define-company',
-      disabled: this.isBusySaving,
-      valid: this.step1Valid
-    },
-    {
-      id: 'step-2-btn',
-      icon: 'mdi-account-multiple-plus',
-      text: 'Add People\nand Roles',
-      to: '/people-roles',
-      disabled: true,
-      valid: this.step2Valid
-    },
-    {
-      id: 'step-3-btn',
-      icon: 'mdi-format-list-text',
-      text: 'Create\nRules',
-      to: '/rules',
-      disabled: true,
-      valid: this.step3Valid
-    },
-    {
-      id: 'step-4-btn',
-      icon: 'mdi-file-document-box-multiple',
-      text: 'Create\nMemorandum',
-      to: '/memorandum',
-      disabled: true,
-      valid: this.step4Valid
-    },
-    {
-      id: 'step-5-btn',
-      icon: 'mdi-file-document-box-check-outline',
-      text: 'Review\nand Confirm',
-      to: '/review-confirm',
-      disabled: this.isBusySaving,
-      valid: this.isApplicationValid
-    }]
-  }
-
-  // Step configuration for fallback
-  private get StepsElse (): Array<any> {
-    return [{
-      id: 'step-1-btn',
-      icon: 'mdi-domain',
-      text: 'Define Your\nCompany',
-      to: '/define-company',
-      disabled: false
-    },
-    {
-      id: 'step-2-btn',
-      icon: 'mdi-dots-horizontal',
-      disabled: true
-    },
-    {
-      id: 'step-3-btn',
-      icon: 'mdi-dots-horizontal',
-      disabled: true
-    },
-    {
-      id: 'step-4-btn',
-      icon: 'mdi-dots-horizontal',
-      disabled: true
-    },
-    {
-      id: 'step-5-btn',
-      icon: 'mdi-dots-horizontal',
-      disabled: true
-    }]
-  }
+  @Getter getSteps!: GetterIF
```

Comment:
This is a big change!

Reply:
Yes :) IMHO actions and stepper should have access to the same steps. So it is better that it is in a common place. Also the stepper should just display and bind the properties according to the data passed to it

Reply:
Absolutely, i think it's a great idea!